RFCs/RFC-0: Lightweight RFC process.md

```diff
@@ -0,0 +1,78 @@
+# RFC-0: Lightweight RFC Process for riff
+
+**To be Reviewed By:** Nov 15th 2019
```

Comment:
is three days typical? Is that too short?

Reply:
That would depend on the scope of the RFC. Ideally, we should have a community meeting every week where we can discuss any Active RFCs and then vote on approving them.

Reply:
removed timeline/deadline from proposal.